mantle/kola/harness.go

```diff
@@ -790,10 +791,24 @@ func runTest(h *harness.H, t *register.Test, pltfrm string, flight platform.Flig
 		}
 		defer in.Close()
 		for _, mach := range tcluster.Machines() {
-			remotepath := fmt.Sprintf("/usr/local/bin/kola-runext-%s", filepath.Base(t.ExternalTest))
+			unit := fmt.Sprintf("kola-runext-%s", filepath.Base(t.ExternalTest))
+			remotepath := fmt.Sprintf("/usr/local/bin/%s", unit)
 			if err := platform.InstallFile(in, mach, remotepath); err != nil {
 				h.Fatal(errors.Wrapf(err, "uploading %s", t.ExternalTest))
 			}
+			defer func(mach platform.Machine) {
```

Comment:
Ahhh OK, so this is running at the end of the test. Which... yeah I guess that works. I was thinking more of processing the journal data we've already pulled out, so we're only querying the journal once and e.g. can still have something even if the node can't be SSH'ed in anymore. But meh... this is fine too (and we still have the primary journal to debug if that happens).

Reply:
Notably if the machine isn't SSH'able here then I doubt that there'd be a primary journal either (that is typically captured during the `platform.Machine`'s destructor).